static/docs/command-reference/get.md

```diff
@@ -95,26 +96,36 @@ is found, that specifies `model.pkl` in its outputs (`outs`). DVC then
 its
 [config file](https://github.com/iterative/example-get-started/blob/master/.dvc/config)).
 
-> A recommended use for downloading binary files from DVC repositories, as done
-> in this example, is to place a ML model inside a wrapper application that
-> serves as an [ETL](https://en.wikipedia.org/wiki/Extract,_transform,_load)
-> pipeline or as an HTTP/RESTful API (web service) that provides predictions
-> upon request. This can be automated leveraging DVC with
+> A recommended use for obtaining binary files from DVC repositories, as done in
+> this example, is to place a ML model inside a wrapper application that serves
+> as an [ETL](https://en.wikipedia.org/wiki/Extract,_transform,_load) pipeline
+> or as an HTTP/RESTful API (web service) that provides predictions upon
+> request. This can be automated leveraging DVC with
 > [CI/CD](https://en.wikipedia.org/wiki/CI/CD) tools.
 
 The same example applies to raw or intermediate <abbr>data artifacts</abbr> as
-well, of course, for cases where we want to download those files or directories
+well, of course, for cases where we want to obtain those files or directories
 and perform some analysis on them.
 
+## Examples: Retrieve a file from a git repository
+
+We can also use `dvc get` to retrieve any file or directory that exists in a git
+repository.
+
+```dvc
+$ dvc get https://github.com/schacon/cowsay/install.sh install.sh
+$ ls
+install.sh
+```
+
 ## Example: Compare different versions of data or model
 
 `dvc get` has the `--rev` option, to specify which version of the repository to
-download a <abbr>data artifact</abbr> from. It also has the `--out` option to
-specify the file or directory path and file name for the download. Combining
-these two options allows us to do something we can't achieve with the regular
-`git checkout` + `dvc checkout` process – see for example the
-[Get Older Data Version](/doc/get-started/older-versions) chapter of our _Get
-Started_ section.
+obtain a <abbr>data artifact</abbr> from. It also has the `--out` option to
+specify the target path. Combining these two options allows us to do something
```

Comment:
Thanks. I agree "target path" is better than before but still a little ambiguous. How about something like "location to place the artifact in the <abbr>workspace</abbr>"?